docs/maatregelen/0-org-03-verken_normen_en_standaarden.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Verken normen en standaarden 
```

Comment:
We moeten denk ik even wachten met deze maatregel publiceren. Nout is bezig met een algemeen stuk tekst over standaarden voor in het Algoritmekader. 
@noutvandeijck : Hoewel nog beknopt, zie hieronder al wat informatie over standaarden.